src/codegen/codegen_neuron_cpp_visitor.hpp

```diff
@@ -492,6 +492,26 @@ class CodegenNeuronCppVisitor: public CodegenCppVisitor {
     void print_global_function_common_code(BlockType type,
                                            const std::string& function_name = "") override;
 
+    /**
+     * Prints setup code for entrypoints from NEURON.
+     *
+     * The entrypoints typically receive a `sorted_token` and a bunch of other things, which then
+     * need to be converted into the default arguments for functions called (recursively) from the
+     * entrypoint.
+     *
+     * This variation prints the fast entrypoint, where NEURON is fully initialized and setup.
+     */
+    void print_entrypoint_setup_code_from_memb_list();
+
+
+    /**
+     * Prints setup code for entrypoints NEURON.
+     *
+     * See `print_entrypoint_setup_code`. This variation should be used when one only has
```

Comment:
Should this be `print_entrypoint_setup_code_from_memb_list`? I don't find any instances of `print_entrypoint_setup_code`...

Reply:
ty, that was it's old name.